interface/forms/note/new.php

```diff
@@ -26,95 +27,82 @@
 /* name of this form */
 $form_name = "note";
 ?>
-
-<html><head>
-
-<?php Header::setupHeader('datetime-picker'); ?>
-
-<script>
-// required for textbox date verification
-var mypcc = <?php echo js_escape($GLOBALS['phone_country_code']); ?>;
-</script>
-
+<html>
+<head>
+    <?php Header::setupHeader('datetime-picker'); ?>
+    <script>
+        // required for textbox date verification
+        const mypcc = <?php echo js_escape($GLOBALS['phone_country_code']); ?>;
+
+        $(function () {
+            $('.datepicker').datetimepicker({
+                <?php $datetimepicker_timepicker = true; ?>
+                <?php $datetimepicker_showseconds = true; ?>
+                <?php $datetimepicker_formatInput = true; ?>
```

Comment:
The above line should be set to false since need to internationalize the date that php brings into the datepicker for it to work (or else it will break).

Reply:
Is it all lines or just line 41?

Reply:
hi @juggernautsei , just line 41, the formatting